setup.cfg

```diff
@@ -1,3 +1,3 @@
 [pycodestyle]
-ignore = E402
-max-line-length = 80
+ignore = E121, E123, E126, E226, E24, E704, W503, E402
```

Comment:
E121, E123, E126, E226, E24, E704, W503 are ignored by pycodestyle by default. Specifying E402 here overwrites the defaults instead of extending them.